crates/pink/src/contract.rs

```diff
@@ -16,17 +16,44 @@ pub struct ExecError {
     message: String,
 }
 
+#[derive(Debug)]
+struct Hooks {
+    on_block_end: Option<u32>,
```

Comment:
Maybe add a comment on what the `u32` is

Reply:
Renamed the Hooks to HookSelectors.